1.8/usage/jobs/getting-started.md

```diff
@@ -0,0 +1,120 @@
+---
+post_title: Getting Started
+nav_title: Getting Started
+menu_order: 10
+---
+
+You can create and administer jobs in the DC/OS web interface, the DC/OS CLI, or via the API. You can add permissions to your job via an access control list (ACL). [Learn more](/1.8/administration/id-and-access-mgt/permissions/service-acls).
+
+# DC/OS Web Interface
+
+## Add a Job
+
+From the DC/OS web interface, click the **Jobs** tab, then the **New Job** button. Fill in the following fields, or toggle to JSON mode to edit the JSON directly:
+
+* **CPU**
+* **Memory**
+* **Disk space**
+* **Command** - The command your job will execute. Leave this blank if you will use a Docker image.
+* **Schedule** - Specify the schedule in chron format, as well as the time zone and starting deadline.
+* **Docker Container** - Fill in this field if you will use a Docker image to specify the action of your job.
+* **Labels**
+
+## Modify, View, or Remove a Job
+
+From the "Jobs" tab, click the name of your job to modify or delete your job. While the job is running you can click the job instance to drill down to "Details," "Files," and "Logs" data. 
+
+# DC/OS CLI
+
+You can create and manage jobs from the DC/OS CLI using `dcos job` commands. To see a full list of available commands, run `dcos job`.
+
+## Add a Job
+
+1. Create a job file in JSON format:
+
+    ```json
+    {
+      "id": "myjob",
+      "description": "A job that sleeps regularly",
+      "run": {
+        "cmd": "sleep 20000",
+        "cpus": 0.01,
+        "mem": 32,
+        "disk": 0
+      },
+      "schedules": [
+        {
+          "id": "sleep-schedule",
+          "enabled": true,
+          "cron": "20 0 * * *",
+          "concurrencyPolicy": "ALLOW"
+        }
+      ]
+    }
+    ```
+    
+    **Note:** You can only assign one schedule to a job.
+  
+  1. Add the job:
+    ```
+    dcos job add <job-id>.json
```

Comment:
the name of the file does not have to be `job-id`.  using job-id probably makes sense but the name is arbitrary.